Core/scripts/dirac-install.py

```diff
@@ -741,26 +741,30 @@ def getTarsLocation(self, project, module=None):
       return S_OK(sourceUrl)
```

Comment:
@zmathe Can you take a look at this please? It makes it so that DIRAC install understands a `install.cfg` such as:
```
LocalInstallation {
  Project = DIRAC
  Release = v7r0p9
  SkipCAChecks = True
  SkipCADownload = True
  DIRACOS = http://lhcb-rpm.web.cern.ch/lhcb-rpm/dirac/DIRACOS111/
}
```

Reply:
Hum, that looks very much like what I asked at the time, and was supposedly implemented no ? https://github.com/DIRACGrid/DIRAC/issues/3517

Reply:
@chaen This is what we use for LHCb http://lhcb-rpm.web.cern.ch/lhcb-rpm/lhcbdirac/lhcb.cfg
As you can see DIRACOS repository is taken from outside installation.

Reply:
Exactly so it should work for @chrisburr use case no ?

Reply:
That doesn't work for the local installation as the local `install.cfg` is loaded first (and has to be so you know what to install). The global configuration files are then loaded and override the local configuration.

Reply:
Then it's a bug, the local configuration should take precedence !

Reply:
According to the code the global installation loaded first: 
https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L1745
and after https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L1752

Reply:
anyway both configuration is loaded and merged. We have DIRACOS in global and local and we take DIRACOS from global ....
If we have diracos in local, we should take from there. This is the problem

Reply:
the following method is wrong: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L1752
This needs to be fixed ...

Reply:
it is fixed in https://github.com/DIRACGrid/DIRAC/pull/4411/commits/3bb73be17cb5b16e285fe7741227ccf931876921

Reply:
This version now accepts the configuration when given in this form:
```
DIRACOS = http://lhcb-rpm.web.cern.ch/lhcb-rpm/dirac/DIRACOS111/
LocalInstallation {
  Project = DIRAC
  Release = v7r0p9
  SkipCAChecks = True
  SkipCADownload = True
}
```